simtbx/nanoBragg/nanoBragg.cpp

```diff
@@ -384,6 +390,7 @@ nanoBragg::init_defaults()
     Xclose=NAN;Yclose=NAN;close_distance=NAN;
     Fclose=NAN;Sclose=NAN;
     ORGX=NAN;ORGY=NAN;
+    detector_is_righthanded=true;
```

Comment:
It may be worth defining what right handed means here.  Namely, if the panel's normal vector (fast cross slow) is in the direction of expected propagation of photons through the material, then the detector is 'right handed'.